.github/workflows/validate-pr.yaml

```diff
@@ -16,10 +20,10 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+          persist-credentials: false
 
       - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.0.1
-
+        uses: helm/chart-testing-action@b0d4458c71155b54fcf33e11dd465dc923550009 # v2.0.1
```

Comment:
When pinning I didn't alter any versions, even if the version appeared to be a typo (all the other workflows use v2.1.0 not v2.0.1)